packages/cli/src/commands/init/init.ts

```diff
@@ -219,11 +234,15 @@ async function createFromTemplate({
       loader.succeed('Dependencies installation skipped');
     }
   } catch (e) {
+    if (e instanceof Error) {
+      logger.error(e.message);
+    }
     loader.fail();
-    throw e;
```

Comment:
This fixes CLI exiting when cocoapods installation fails

Reply:
Maybe something like: 

Installing pods failed. This doesn't affect project initialization and you can safely proceed. However, you will need to install pods manually when running iOS, follow additional steps in "Run instructions for iOS" section. 

To avoid duplicating the steps

Reply:
👍🏼